src/metabase/models/setting/cache.clj

```diff
@@ -76,8 +76,8 @@
   ;; for MySQL, cast(current_timestamp AS char); for H2 & Postgres, cast(current_timestamp AS text)
   (let [current-timestamp-as-string-honeysql (h2x/cast (if (= (mdb.connection/db-type) :mysql) :char :text)
                                                        [:raw "current_timestamp"])]
-    ;; attempt to UPDATE the existing row. If no row exists, `update-where!` will return false...
-    (or (db/update-where! 'Setting {:key settings-last-updated-key} :value current-timestamp-as-string-honeysql)
+    ;; attempt to UPDATE the existing row. If no row exists, `t2/update!` will return 0...
+    (or (pos? (t2/update! :setting  {:key settings-last-updated-key} {:value current-timestamp-as-string-honeysql}))
```

Comment:
triggered during `pre-update`, so need to call with table name to avoid hooks.

Reply:
Should this use `(t2/table-name Setting)` instead of `:setting`?

Reply:
IIRC there was some dep cyclic problem with using `Setting`.

But nevertheless, they're the same so I'll leave it.